pkg/util/paramtable/autoindex_param.go

```diff
@@ -260,18 +260,18 @@ func setDefaultIfNotExist(params map[string]string, key string, defaultValue str
 }
 
 const (
-	FloatVectorDefaultMetricType       = metric.COSINE
+	VectorDefaultMetricType            = metric.COSINE
 	SparseFloatVectorDefaultMetricType = metric.IP
```

Comment:
I know you are trying to find out a common type for int/float. I guess `DenseNumericVector` is a not bad one. You can also apply it to all other places

Reply:
Use FloatVectorDefaultMetricType and IntVectorDefaultMetricType seperately